BranchUnityTestBed/Assets/Branch/Editor/BranchEditor.cs

```diff
@@ -405,8 +405,6 @@ public class BranchEditor : Editor {
 		// <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
@antonargunov  Can you please  remove these comments  also

Reply:
#120